ndbench-dynamodb-plugins/src/main/java/com/netflix/ndbench/plugin/dynamodb/DynamoDBKeyValue.java

```diff
@@ -287,6 +319,131 @@ private void initializeTable() {
         } catch (InterruptedException e) {
             logger.error("Table interrupted exception: " + e);
         }
+
+        // if autoscaling was disabled, we are finished
+        if (!config.getAutoscaling()) {
+            return;
+        }
+
+        // configure autoscaling - from aws docs
+        // https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/AutoScaling.HowTo.SDK.html
+        final String resourceID = "table/" + config.getTableName();
```

Comment:
These `final` variables definition at the end of the method does not help the readability of the code. Their scope of usage is in `createAndVerifyScalableTargetAndScalingPolicy` hence I am wondering if it is better to define them in there. Moreover, the class is becoming too long, does it worth splitting into a different class?

Reply:
removed the final and moving to a different class.